packages/core/rollup.config.js

```diff
@@ -2,13 +2,14 @@ import commonjs from "@rollup/plugin-commonjs";
 import json from "@rollup/plugin-json";
 import { nodeResolve } from "@rollup/plugin-node-resolve";
 
+import { extractExports } from "../../extractExports.js";
```

Comment:
nit: can we do something about this import, maybe creating package for internal build utils

Reply:
fair

Reply:
That was something I thought we can do in the future. Amazing that you did it here